suzieq/poller/worker/services/routes.py

```diff
@@ -36,6 +36,22 @@ def _common_data_cleaner(self, processed_data, _):
 
         return processed_data
 
+    def _clean_aos_data(self, processed_data, _):
+
+        entry_new = []
+
+        for entry in processed_data:
+
+            entry_dict = {
+                'prefix': entry['dest_addr'],
+                'nexthopIps': [entry['gateway_addr']],
+                'protocol': entry['protocol'],
+            }
```

Comment:
Would it be possible to retrieve also fields like: `oifs`, `source`, `vrf`, `action`?

Reply:
`oifs` could be added, but another more complex command has to be parsed: `show ip router database`